tests/torch/data/reference_graphs/quantized/adjust_paddings/npu_all_activations_int8_requnt.dot

```diff
@@ -2,36 +2,40 @@ strict digraph  {
 "1 MultiBranchesModel/ReLU/relu_0" [id=1, label="relu_#1", style=filled, type=relu];
 "2 MultiBranchesModel/ReLU/AsymmetricQuantizer/asymmetric_quantize_0" [color=green, id=2, label="AFQ_[B:8 M:A SGN:S PC:N]_#2_G0", style=filled, type=asymmetric_quantize];
-"3 MultiBranchesModel/NNCFConv2d[conv_a]/ModuleDict[pre_ops]/UpdateWeight[0]/SymmetricQuantizer[op]/symmetric_quantize_0" [color=green, id=3, label="WFQ_[B:8 M:S SGN:U PC:Y]_#3_G0", style=filled, type=symmetric_quantize];
+"3 MultiBranchesModel/NNCFConv2d[conv_a]/ModuleDict[pre_ops]/UpdateWeight[0]/SymmetricQuantizer[op]/symmetric_quantize_0" [color=green, id=3, label="WFQ_[B:8 M:S SGN:S PC:Y]_#3_G0", style=filled, type=symmetric_quantize];
 "4 MultiBranchesModel/NNCFConv2d[conv_a]/SymmetricQuantizer/symmetric_quantize_0" [color=green, id=4, label="AFQ_[B:8 M:S SGN:U PC:Y]_#4_G0", style=filled, type=symmetric_quantize];
 "5 MultiBranchesModel/NNCFConv2d[conv_a]/conv2d_0" [color=purple, id=5, label="DW_Conv2d_k3x3_PAD_#5", style=filled, type=conv2d];
-"6 MultiBranchesModel/MaxPool2d[max_pool_b]/max_pool2d_0" [id=6, label="max_pool2d_#6", style=filled, type=max_pool2d];
-"7 MultiBranchesModel/NNCFConv2d[conv_b]/ModuleDict[pre_ops]/UpdateWeight[0]/SymmetricQuantizer[op]/symmetric_quantize_0" [color=red, id=7, label="WFQ_[B:4 M:S SGN:U PC:Y]_#7_G0", style=filled, type=symmetric_quantize];
-"8 MultiBranchesModel/NNCFConv2d[conv_b]/conv2d_0" [color=lightblue, id=8, label="conv2d_k5x5_PAD_#8", style=filled, type=conv2d];
-"9 MultiBranchesModel/NNCFConv2d[conv_c]/ModuleDict[pre_ops]/UpdateWeight[0]/SymmetricQuantizer[op]/symmetric_quantize_0" [color=red, id=9, label="WFQ_[B:4 M:S SGN:U PC:Y]_#9_G0", style=filled, type=symmetric_quantize];
-"10 MultiBranchesModel/NNCFConv2d[conv_c]/conv2d_0" [color=lightblue, id=10, label="conv2d_k3x3_PAD_#10", style=filled, type=conv2d];
-"11 MultiBranchesModel/NNCFConv2d[conv_d]/ModuleDict[pre_ops]/UpdateWeight[0]/SymmetricQuantizer[op]/symmetric_quantize_0" [color=red, id=11, label="WFQ_[B:4 M:S SGN:U PC:Y]_#11_G0", style=filled, type=symmetric_quantize];
-"12 MultiBranchesModel/NNCFConv2d[conv_d]/SymmetricQuantizer/symmetric_quantize_0" [color=red, id=12, label="AFQ_[B:4 M:S SGN:U PC:N]_#12_G0", style=filled, type=symmetric_quantize];
-"13 MultiBranchesModel/NNCFConv2d[conv_d]/conv2d_0" [color=lightblue, id=13, label="conv2d_k3x3_#13", style=filled, type=conv2d];
-"14 /nncf_model_output_0" [id=14, label="nncf_model_output_#14", style=filled, type=nncf_model_output];
-"15 /nncf_model_output_1" [id=15, label="nncf_model_output_#15", style=filled, type=nncf_model_output];
-"16 /nncf_model_output_2" [id=16, label="nncf_model_output_#16", style=filled, type=nncf_model_output];
-"17 /nncf_model_output_3" [id=17, label="nncf_model_output_#17", style=filled, type=nncf_model_output];
+"6 MultiBranchesModel/MaxPool2d[max_pool_b]/SymmetricQuantizer/symmetric_quantize_0" [color=green, id=6, label="AFQ_[B:8 M:S SGN:U PC:N]_#6_G0", style=filled, type=symmetric_quantize];
```

Comment:
previously conv had asymmetric int8 activations
![image](https://github.com/openvinotoolkit/nncf/assets/4014476/43d22d2e-b117-4d0f-818a-de9e0acc482f)

Currently NPU config have symmetric activations on the first place, that's why there're 2 FQ.
![image](https://github.com/openvinotoolkit/nncf/assets/4014476/52183166-1134-447b-878b-50ec04ccba20)

Reply:
The same for all `requant` tests

Reply:
Other references just changed assymetric to symmetric in 8bit activations

Reply:
Thanks for the clarification.
But why do we have an asymmetric quantizer after the relu layer?